addons/account_peppol/models/account_edi_proxy_user.py

```diff
@@ -100,7 +100,7 @@ def _cron_peppol_get_new_documents(self):
 
                 enc_key = content["enc_key"]
                 document_content = content["document"]
-                filename = content["filename"]
+                filename = content["filename"] or 'attachment' # default to attachment, which should not usually happen
```

Comment:
we're supposed to receive the filename from the peppol proxy here. But in case the proxy doesn't retrieve the attachment name correctly, `content["filename"]` is False. I think it's not nice to save files as `False.xml` in such cases.